storybook/stories/API/polar/PolarGrid.stories.tsx

```diff
@@ -1,8 +1,46 @@
 import React from 'react';
+import { Args } from '@storybook/react';
 import { PolarGrid, ResponsiveContainer, Surface } from '../../../../src';
 
+const GeneralProps: Args = {
+  cx: {
+    description: 'The x-coordinate of center.',
+    table: { type: { summary: 'number' }, category: 'General' },
+  },
+  cy: {
+    description: 'The y-coordinate of center.',
+    table: { type: { summary: 'number' }, category: 'General' },
+  },
+  innerRadius: {
+    description: 'The radius of the inner polar grid.',
+    table: { type: { summary: 'number' }, category: 'General' },
+  },
+  outerRadius: {
+    description: 'The radius of the outer polar grid.',
+    table: { type: { summary: 'number' }, category: 'General' },
+  },
```

Comment:
Should some of these possibly be categorised `Polar`? And be shared across all Polar components?